roles/ucp/templates/ucp.j2

```diff
@@ -49,17 +49,16 @@ start)
     /usr/bin/docker run --rm -t --name ucp \
         -v /var/run/docker.sock:/var/run/docker.sock \
         -e UCP_ADMIN_USER={{ ucp_admin_user }} -e UCP_ADMIN_PASSWORD={{ ucp_admin_password }} \
+    {% if run_as == "master" -%}
+        docker/ucp join {{ ucp_controller_replica }} --host-address={{ node_addr }} \
```

Comment:
@mapuri yes, so on 3 node master cluster the number of ucp infra containers were not how many they should be. uday from docker team mentioned about the positioning aspect of the --replica flag. I tried out the changes, and it works now.